src/style.css

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.big-form {
+  position: relative;
+  margin-top: 20px;
+  display: block;
+}
+
+.icon {
+  display: block;
+  position: absolute;
+}
+
+.icon--big {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.input--big {
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+}
+
+.input {
+  font-size: 16px;
+  font-family: inherit;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  font-weight: 300;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.small-form {
+  position: relative;
+  display: block;
+}
+
+.input--small {
+  height: 42px;
+  padding-left: 33px;
+}
+
+.input--big::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input--small::placeholder {
+  font-style: normal;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.input--big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input--small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input--big:focus {
+  border-radius: 3px;
+  color: #3d4e61;
+  font-size: 16px;
+  line-height: 22px;
+  text-shadow: (0 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input--small:focus {
+  border-radius: 3px;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 800;
+  font-size: 14px;
+  line-height: 19px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
Move all repeatable colors to CSS variables and move them to :root selector.

[More info. ](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)